src/AppBundle/Entity/User.php

```
@@ -986,6 +995,44 @@ public function getAgreeTermsUseDate()
     }
 
     /**
+     * @return bool
+     */
+    public function getCoDeputyClientConfirmed()
+    {
+        return $this->coDeputyClientConfirmed;
+    }
+
+    /**
+     * @param bool $coDeputyClientConfirmed
+     */
+    public function setCoDeputyClientConfirmed($coDeputyClientConfirmed)
+    {
+        $this->coDeputyClientConfirmed = $coDeputyClientConfirmed;
+    }
+
+    /**
+     * Return true if the client has other users
+     *
+     * @JMS\VirtualProperty
+     * @JMS\Type("boolean")
+     * @JMS\SerializedName("is_co_deputy")
+     * @JMS\Groups({"user"})
+     *
+     * @return bool
+     */
+    public function isCoDeputy()
+    {
+        $isCoDeputy = false;
+        if ($this->isLayDeputy()) {
+            $client = $this->getFirstClient();
+            if (!empty($client)) {
+                $isCoDeputy = count($client->getUsers()) > 1;
+            }
```

Maybe throw an exception if client not set?